modules/serverless-kafka-tls-sasl.adoc

```diff
@@ -0,0 +1,87 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-configuring-tls-authentication-against-an-apache-kafka_{context}"]
+= Configuring authentication for Kafka
+
+In production, Kafka clusters are often secured using the TLS or SASL authentication methods. This section shows how to configure the `KafkaChannel` to work against a protected Kafka cluster using TLS or SASL.
+
+== Configuring TLS authentication
+
+.Prerequisites
+
+* A CA certificate as a `.pem` file.
+* A client certificate and key as `.pem` files.
+
+.Procedure
+
+. Create the certificate files as secrets in your chosen namespace:
++
+----
+$ kubectl create secret --namespace <namespace> generic <kafka-auth-secret> \
+  --from-file=ca.crt=caroot.pem \
+  --from-file=user.crt=certificate.pem \
+  --from-file=user.key=key.pem
+----
++
+Use the key names `ca.crt`, `user.crt`, and `user.key`. Do not change them.
```

Comment:
Make this a note if not doing it would break something maybe?

[IMPORTANT]
====
Use the key names `ca.crt`, `user.crt`, and `user.key`. Do not change them.
====

Reply:
Made into a note. Re "break something" - what it breaks is the command invocation, which accepts those names. Is that correct @matzew ? If so, it's sort of low additional value to explicitly state that, I think (?).

Reply:
those are required keys, like an API - if you mess that up, it wont work.